### PR TITLE
Project Ceaser Cipher Conversion Project

### PR DESCRIPTION
**Pull request by Damini2004**
### Issue Title: Ceaser Cipher Conversion Project

- **Name of the project** : Ceaser Cipher Conversion Project 
- **Category:** Vanilla-JS-Projects
- **Name:** Damini Chachane
- **GitHub ID:**  https://github.com/Damini2004
- **Idenitfy yourself:** GSSOC Contributor

Closes: #124 

### Describe the add-ons or changes you've made 📃

Made a Cipher text language conversion which helps you in changing a ciphertext  into your plain language

### Type of change ☑️

What sort of change have you made:

- [x] New feature (non-breaking change which adds functionality)

### How Has This Been Tested? ⚙️
It has been tested on several text as per its key from ciphertext to plaintext and vice versa.

### Checklist: ☑️

- [x] My code follows the [Contributing Guidelines](https://github.com/Avdhesh-Varshney/WebMasterLog/blob/main/CONTRIBUTING.md) & [Code of Conduct](https://github.com/Avdhesh-Varshney/WebMasterLog/blob/main/CODE_OF_CONDUCT.md) of this project.
- [x] This PR does not contain plagiarized content.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly wherever it was hard to understand.
- [x] My changes generate no new warnings.

### Screenshots 📷
dark
light

### Note to reviewers 📄
Kindly go throught it

### CODE REVIEW

File: Vanilla-JS-Projects/Intermediate/Ceaser-Cipher/light.webp
Comment (Avdhesh-Varshney) [inline Vanilla-JS-Projects/Intermediate/Ceaser-Cipher/light.webp:1]: Rename this file with `screenshot` name. 
And don't forget to update README file from `light.webp` to `screenshot.webp` file.

**Review by Avdhesh-Varshney — CHANGES_REQUESTED** [review]
@Damini2004

**Review by Avdhesh-Varshney — APPROVED** [review]
@Damini2004 PR Approved 🎉
But still you have not linked you issue with the PR. But i have linked it so take a review on it.

**Comment by Damini2004** [comment]
Kindly add GSSOC 24 label that will be more beneficial

**Pull request merged**